pyiceberg/table/__init__.py

```diff
@@ -1673,12 +1673,8 @@ def refs(self) -> Dict[str, SnapshotRef]:
         return self.metadata.refs
 
     def _do_commit(self, updates: Tuple[TableUpdate, ...], requirements: Tuple[TableRequirement, ...]) -> None:
-        response = self.catalog._commit_table(  # pylint: disable=W0212
-            CommitTableRequest(
-                identifier=TableIdentifier(namespace=self._identifier[:-1], name=self._identifier[-1]),
-                updates=updates,
-                requirements=requirements,
-            )
+        response = self.catalog.commit_table(  # pylint: disable=W0212
```

Comment:
```suggestion
        response = self.catalog.commit_table(
```

i think we can get rid of disable pylint here since its no longer a private method
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/protected-access.html

Reply:
Good one, thanks!